### ISSUE TITLE
Use database volumes in docker-compose by default

### ISSUE THREAD

**Issue by GaryPWhite** [chaoss/augur#2676]
**Is your feature request related to a problem? If so, please describe the problem:**
When installing / using augur with docker -- the postgres database is tied directly to the data within the encircling container. As containers / drives sometimes fail -- this is a brittle way to set up the database that may hold 100's of thousands of repositories worth of valuable metadata.

**Potential solutions:**
mount a volume into the postgres docker image that could be mounted from a volume, eventually put in a redundant filesystem / nfs mount or standard backup mount on a cloud instance. 

From their documentation the mount would be associated with a respective `PGDATA` environment variable to avoid unexpected behavior.
```
$ docker run -d \
	--name some-postgres \
	-e POSTGRES_PASSWORD=mysecretpassword \
	-e PGDATA=/var/lib/postgresql/data/pgdata \
	-v /custom/mount:/var/lib/postgresql/data \
	postgres

```

This allows a decoupled volume to exist regardless of containers being started/restarted/removed/recreated.

**Additional context:**
[postgres docker image](https://hub.docker.com/_/postgres/)

I'm happy to work on this, just wanted to see the interest or if there's a good reason not to include the data in a volume!

**Comment by sgoggins** [comment]
@GaryPWhite : That's a really good idea!  We would quickly merge a PR if you have time to do that!

**Comment by GaryPWhite** [comment]
will do!

**Comment by sgoggins** [comment]
Fixed by PR #2677
